plugin/server/metaRunners/MRPLUGIN_UPackPack.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    https://inedo.com/support/documentation/proget/reference/upack
+
+   upack pack «source» 
+        [\-\-targetDirectory=«targetDirectory»]
```

Comment:
I've been using `==` instead of `\-\-` for the double-dashes. Also using `<` and `>` rather than `«` and `»` so the file can be easily run through HTML Tidy with XML settings and it won't corrupt the doc. Can you update?

Reply:
(I fixed it on the previous submission post-pull.)

Reply:
Sure.  I'll get right on it.